src/components/SlateEditor/plugins/details/onDelete.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { Block, Document, Editor, Inline, Node, Point } from 'slate';
+
+type ParentNode = Document | Block | Inline | null;
+
+const shouldDeleteIfInsideOfDetailsblock = (
```

Comment:
Er dette samme funksjon som i den andre fila? I tilfelle burde dei heller dras ut i en egen util.js

Reply:
Ikke helt, `onDelete` sjekker på next-node for å ikke slette ting utenfor details osv. mens `onBackspace` sjekker på previous-node, så litt andre sjekks må gjøres der. Funksjonene ligner på hverandre men funksjonaliteten er annerledes.